rpm/cairo.spec

```diff
@@ -1,24 +1,25 @@
 Name:       cairo
 
 Summary:    A vector graphics library
-Version:    1.17.4
+Version:    1.17.8
 Release:    1
 License:    LGPLv2 or MPLv1.1
 URL:        http://www.cairographics.org
 Source0:    %{name}-%{version}.tar.xz
 Requires(post): /sbin/ldconfig
 Requires(postun): /sbin/ldconfig
+BuildRequires:  meson
 BuildRequires:  pkgconfig(libpng)
 BuildRequires:  pkgconfig(freetype2) >= 9.7.3
 BuildRequires:  pkgconfig(fontconfig) >= 2.2.95
-BuildRequires:  pkgconfig(pixman-1) >= 0.30.0
+BuildRequires:  pkgconfig(pixman-1) >= 0.36.0
 BuildRequires:  pkgconfig(glib-2.0)
 BuildRequires:  pkgconfig(gobject-2.0)
 BuildRequires:  pkgconfig(zlib)
 BuildRequires:  pkgconfig(libxml-2.0)
 BuildRequires:  libGLESv2-devel
 BuildRequires:  libEGL-devel
```

Comment:
You can now remove these build requirements because gl backend was removed.

Reply:
Ah... there is always something that gets away.